### ISSUE TITLE
"seperater" should be "separator"

### ISSUE THREAD

**Issue by bgentry** [scurker/currency.js#10]
You've got an option `seperater` for specifying the string that should separate groups of numbers (i.e. `,`).  That's spelled incorrectly, and should instead be `separator`.

I'd be happy to submit a PR with this change. Would you like to leave the old option around without documentation for compatibility reasons?

**Comment by scurker** [comment]
D'oh. I let spelling get the best of me. I'll need to update the tests to reflect any changes as well as deprecating the wrong spelling. I'll try to take a look at this in the next couple of days.

**Comment by scurker** [comment]
This should now be fixed with [v0.3.2](https://github.com/scurker/currency.js/commit/bef8aa8b8d945bb80341c003a2b163e6fe3d1a89), thanks!

**Comment by bgentry** [comment]
@scurker Thanks! Just a heads up it does look like you also need to update the bower.json manifest to reflect the new version when you release in the future. This warning gets generated now:

```
bower currency.js#0.3.2       mismatch Version declared in the json (0.3.0) is different than the resolved one (0.3.2)
```

**Comment by scurker** [comment]
@bgentry Thanks for the heads up. I resolved the version issue as well as a few of the package dependencies that weren't in the bower repository. The versions should now stay in sync with future releases.
